webapp/channels/src/actions/views/drafts.ts

```diff
@@ -111,7 +111,14 @@ export function updateDraft(key: string, value: PostDraft|null, rootId = '', sav
             const connectionId = getConnectionId(state);
             const userId = getCurrentUserId(state);
             try {
-                await upsertDraft(updatedValue, userId, rootId, connectionId);
+                if (value?.message === '' || value?.message.replace(/\s/g, '').length || (value && value?.fileInfos.length > 0)) {
+                    if (value?.message.replace(/\s/g, '').length) {
```

Comment:
i have few questions here
1. Istead of two checks `value?.message === '' || value?.message.replace(/\s/g, '').length` whats stopping us from using only one check `value.message.trim().length`
2. Instead of this if else ladder, can we just create a new message variable just trims the value.message